src/resources/raids/policestation.ts

```diff
@@ -37,8 +37,8 @@ export const station: Location = {
 		},
 		{
 			type: 'LootChannel',
-			name: 'food-court',
-			display: 'Food Court',
+			name: 'Evidence Locker',
```

Comment:
names must follow discord channel naming rules (lowercase and use - instead of spaces)
```suggestion
			name: 'evidence-locker',
```